docs/API/Elements.md

```diff
@@ -253,6 +279,15 @@ Any element property can be used to filter by using the <code>@</code> operator.
 var big = a.findOne('..(@size==10)');
 ```
 
+Here is an example of searching for a child with a name set in the inspector
```

Comment:
For stylistic purposes, you should add a space between "Here is an example..." and the code above it.

Additionally, add a period after "inspector" to keep it in line with the rest of the text in the doc.